circle.yml

```diff
@@ -11,10 +11,12 @@ checkout:
     - chmod a+x ./circle_setup_rvm.sh
     - sudo -E ./circle_setup_rvm.sh
     - sudo -i /home/ubuntu/content_delivery_system/CDS/install.sh -y
+    - cd CDS/js; npm install
```

Comment:
Circle provide a slightly cleaner way of doing this so we don't need the cd https://github.com/guardian/flexible-content/blob/master/circle.yml#L60